__tests__/app.test.js

```diff
@@ -782,4 +782,36 @@ describe("204: delete comment by comment_id", () => {
         });
     });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api serves up a json representation of all the available endpoints of the api",()=>{
+  test("200: responds with an object with the following keys to represent endpoints",()=>{
+    return request(app)
+    .get("/api")
+    .expect(200)
+    .then(({ body }) => {
+
+      const api = body.api;
+
+      expect(Object.keys(api)).toHaveLength(11);
+
+      
+        expect(api).toEqual(
+          expect.objectContaining({
```

Comment:
Great use of objectContaining to keep tests flexible in case you want to add any more endpoints later